### DISCUSSION TITLE
How Automatic update work with DistroAV / OBS-NDI

### DISCUSSION THREAD

**Discussion by Trouffman** [DistroAV/DistroAV#1118] [Show and tell]
Here is a quick overview of how an automatic update managed by the plugin look-like on v.4.14.1 onwards.

At OBS launch, it will check if there is a new plugin update.

![image](https://github.com/user-attachments/assets/d35ba71a-5ede-4bd5-8539-16397f2adeb6)

You can check for update Manually in the plugin main settings (Tools > NDI Output Settings > Check for update)

![image](https://github.com/user-attachments/assets/6b77682f-c4f0-40d1-aff1-5367fec50ef7)

### Question
"OBS crashed or I clicked cancel and now i don't get the prompt for the new update when launching OBS"
The Update check is happening only every few hours to avoid overloading servers and being invasive.
To get the automatic update, Manually check for Update (Tools > NDI Output Settings > Check for update)

**Comment by Barcaboss** [comment]


**Reply by Trouffman** [reply]
Please share the OBS Log file. In it's own issue.

This means there are some leftover of OBS-ndi.
